diagnostic.md

```diff
@@ -5,74 +5,84 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The backend's role is to persist data used by you application.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+The model is used to fetch data.
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller communicates with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+The view is the UI interface.  We use the MVC pattern using curl type request
+to simulate what the UI would send us.
```

Comment:
Close, we manage our view states on the front-end. This lessens the amount of requests we have to make to the server. Very SPA-like.